src/Application/Redux/reducers.js

```diff
@@ -70,7 +70,15 @@ function trumpMM(state = StateUtils.getInitialState(), action) {
       return Object.assign({}, state, { news: state.news.addArticle(bounceArticle) });
 
     case UPGRADE_DENOMINATION:
-      return Object.assign({}, state, { mint: state.mint.toNextDenomination });
+      let mintDelta;
+      let newsArticle;
+      if (state.bank.cash >= state.mint.nextDenomination.priceToUnlock) {
+        mintDelta = { mint: state.mint.toNextDenomination };
+        newsArticle = `Trump's greatness has improved the value of US currency!`;
+      } else {
+        newsArticle = `Trump's experiments in alchemy have failed to produce results`;
```

Comment:
Although these are funny and I like them...I'm concerned that Vee's feedback suggests that our humor gets in the way of people understanding the 'Appflow'.